### ISSUE TITLE
feat(prisma): upgrade to v5.15.0

### ISSUE THREAD

**Issue by jordancparker** [RobertCraigie/prisma-client-py#969]
Hi, 

Would it be possible to bump the version to v5.15.0 as there is the benefit of the below preview feature that will help organise folders. 

```
generator client {
  provider        = "prisma-client-js"
  previewFeatures = ["prismaSchemaFolder"]
}
```

**Comment by ijpiantanida** [comment]
Prisma recently released v5.17.0 to [address some performance issues](https://github.com/prisma/prisma/issues/23573#issuecomment-2230518040).

It'd be nice to get that version in the PY client as well.

**Comment by RobertCraigie** [comment]
`v0.14.0` has been released which bumps the prisma version to `v5.17.0`!
